extensions/amp-viewer-integration/0.1/test/test-highlight-handler.js

```diff
@@ -409,57 +412,79 @@ describes.realWin.skip(
-        env.sandbox.stub(platform, 'getMajorVersion').returns(93);
-        let whenFirstVisiblePromiseResolve;
-        const whenFirstVisiblePromise = new Promise((resolve) => {
-          whenFirstVisiblePromiseResolve = resolve;
+      .run('with Chrome >= 93', () => {
+        let root = null;
+
+        beforeEach(() => {
+          const {document} = env.win;
+          root = document.createElement('div');
+          document.body.appendChild(root);
+          const div0 = document.createElement('div');
+          div0.textContent = 'text in amp doc';
+          root.appendChild(div0);
+          const div1 = document.createElement('div');
+          div1.textContent = 'highlighted text';
+          root.appendChild(div1);
+
+          //  Used in Chrome 93+
+          env.sandbox.stub(env.ampdoc, 'whenFirstVisible');
+
+          const platform = Services.platformFor(env.ampdoc.win);
+          if (platform.isChrome()) {
+            env.sandbox.stub(platform, 'getMajorVersion').returns(93);
+          }
         });
-        env.sandbox
-          .stub(ampdoc, 'whenFirstVisible')
-          .returns(whenFirstVisiblePromise);
 
-        const highlightHandler = new HighlightHandler(ampdoc, {
-          sentences: ['amp', 'highlight'],
-        });
+        it('should highlight using text fragments', async () => {
+          const {ampdoc} = env;
+          let whenFirstVisiblePromiseResolve;
+          const whenFirstVisiblePromise = new Promise((resolve) => {
+            whenFirstVisiblePromiseResolve = resolve;
+          });
+          ampdoc.whenFirstVisible.returns(whenFirstVisiblePromise);
 
-        const updateUrlWithTextFragmentSpy = env.sandbox.spy();
-        highlightHandler.updateUrlWithTextFragment_ =
-          updateUrlWithTextFragmentSpy;
+          const highlightHandler = new HighlightHandler(ampdoc, {
+            sentences: ['amp', 'highlight'],
+          });
 
-        whenFirstVisiblePromiseResolve();
-        await whenFirstVisiblePromise;
+          const updateUrlWithTextFragmentSpy = env.sandbox.spy();
+          highlightHandler.updateUrlWithTextFragment_ =
+            updateUrlWithTextFragmentSpy;
 
-        expect(updateUrlWithTextFragmentSpy).to.be.calledOnce;
-        expect(updateUrlWithTextFragmentSpy.getCall(0).args[0]).to.equal(
-          'text=amp&text=highlight'
-        );
-      });
+          whenFirstVisiblePromiseResolve();
+          await whenFirstVisiblePromise;
 
-    // TODO(dmanek): remove `ifChrome` once we remove Chrome version detection
-    it.configure()
-      .ifChrome()
-      .run(
-        'should not highlight using text fragments for Chrome 92',
-        async () => {
+          expect(updateUrlWithTextFragmentSpy).to.be.calledOnce;
+          expect(updateUrlWithTextFragmentSpy.getCall(0).args[0]).to.equal(
+            'text=amp&text=highlight'
+          );
+        });
+
+        it('should not highlight using text fragments for Chrome 92', async () => {
```

Comment:
Is this test nested within the "Chrome 93" suite? Something is off about that

Reply:
Renamed the test suite.